4.analyze_data/notebooks/linear_model/nbconverted/0.fit_linear_model_strat.py

```diff
@@ -38,14 +38,28 @@
 # In[3]:
 
 
+# # Filter by failing hearts and specific treatments
+# specific_type = ["DMSO"]
+# specific_cell_types = ["failing", "healthy"]
+
+# filtered_df = data_df[
+#     (data_df['Metadata_treatment'].isin(specific_type)) &
+#     (data_df['Metadata_cell_type'].isin(specific_cell_types))
+# ]
```

Comment:
Ah great catch! I had to change up the code only for the failing drug_x versus healthy DMSO because these are the only combos I could have. I'll either remove or figure out how to change this to keep it in there.